image-filters/image-filters.markdown

```diff
@@ -667,25 +664,26 @@ picture.
 Once we’ve extracted the "dominant" hue, we can choose to either show or hide
 it in the image. We can show the dominant hue with varying tolerance (ranges
 around it that we will accept). Pixels that don’t fall into this range can be
-changed to grayscale, by setting the value based on the brightness.
+changed to grayscale by setting the value based on the brightness.
 \aosafigref{500l.imagefilters.showdominant} shows the dominant hue determined
 using a hue range of 240, and with varying tolerance. The tolerance is the
```

Comment:
i think part of the problem is that I'm using different words for "hue range" / "number of buckets" - should probably clarify that number of buckets == hue range, which is the colorMode argument.